Odysee/Controllers/Content/FileViewController.swift

```diff
@@ -470,13 +471,11 @@ class FileViewController: UIViewController, UIGestureRecognizerDelegate, UINavig
                     thumbnailDisplayUrl = URL(string: singleClaim.value!.thumbnail!.url!)!
                 }
                 contentInfoImage.pin_setImage(from: thumbnailDisplayUrl)
-                let tmpImageView = UIImageView()
-                tmpImageView.pin_setImage(from: contentUrl, processorKey: String(format: "iv_%@", singleClaim.claimId!)) { (result, unsafePointer) -> UIImage? in
-                    guard let image = result.image else { return nil }
-                    DispatchQueue.main.async {
+                PINRemoteImageManager.shared().downloadImage(with: contentUrl!) { result in
```

Comment:
Great! This is exactly what I was looking for, but I couldn't find it in https://github.com/pinterest/PINRemoteImage/blob/master/README.md. Would be nice to add it as an example.